code/modules/paperwork/photography.dm

```diff
@@ -199,8 +225,20 @@
 		atoms.Add(the_turf);
 		// As well as anything that isn't invisible.
 		for(var/atom/A in the_turf)
-			if(A.invisibility) continue
+			var/obj/effect/spookyghost/SPOOK
+			if(A.invisibility )
+				if(see_ghosts && istype(A,/mob/dead/observer))
+					var/mob/dead/observer/O = A
+					if(O.following) //so you dont see ghosts following people like antags, etc.
+						continue
+					else
+						SPOOK = new(A.loc)
+						A = SPOOK
+				else
+					continue
 			atoms.Add(A)
+			if(SPOOK)

```

Comment:
theres got to be a really simple thing to this whole sprite issue..and i am not seeing it...